src/vm/gccover.h

```diff
@@ -108,6 +107,75 @@ class GCCoverageInfo {
 
 #endif // _TARGET_*
 
+// The body of this method is in this header file to allow
+// mscordaccore.dll to link without getting an unsat symbol
+//
+inline bool IsGcCoverageInterruptInstructionVal(UINT32 instrVal)
+{
+#if defined(_TARGET_ARM64_)
+
+    switch (instrVal)
+    {
+    case INTERRUPT_INSTR:
+    case INTERRUPT_INSTR_CALL:
+    case INTERRUPT_INSTR_PROTECT_RET:
+        return true;
+    default:
+        return false;
+    }
+
+#elif defined(_TARGET_ARM_)
+
+    size_t instrLen = GetARMInstructionLength(instrVal);
```

Comment:
To be explicit about the type mismatch here, I'd like to see an explicit cast, like:
```
// Use the first 2 bytes to determine the actual instruction size.
size_t instrLen = GetARMInstructionLength((WORD)instrVal);
```

Reply:
OK, I will move this line (133) up here to cover this case:

` UINT16 instrVal16 = static_cast<UINT16>(instrVal);`